rasa/utils/plotting.py

```diff
@@ -14,10 +15,20 @@
 
 
 def _fix_matplotlib_backend() -> None:
-    """Tries to fix a broken matplotlib backend..."""
+    """Tries to fix a broken matplotlib backend."""
+    try:
+        backend = matplotlib.get_backend()
+    except Exception:  # skipcq:PYL-W0703
+        logger.error(
+            "Cannot retrieve Matplotlib backend, likely due to a compatibility "
+            "issue with system dependencies. Please refer to the documentation: "
+            "https://matplotlib.org/stable/tutorials/introductory/usage.html#backends"
+        )
+        raise
```

Comment:
should we even raise a RasaException here?

Reply:
Good question 🤔 I'd prefer to keep visibility about dependency issues to be honest - even this one. I'd rather ignore this one within Sentry.

Reply:
I agree!